__tests__/Fuzzy.test.ts

```diff
@@ -0,0 +1,50 @@
+// ./__tests__/Fuzzy.test.ts
+//
+// Unittests for the Fuzzy class.
+
+// Vitest essential imports.
+import { describe, expect, it } from 'vitest';
```

Comment:
lets add this to [globals](https://vitest.dev/config/#globals) instead so we need not import for every test

Reply:
@ad1992 
- [x] Removed redundant import in 9bfc05c.